### PR TITLE
Create a sticky TimeTracker live_view

### LINKED ISSUE #317: A menu for the TimeTracker in the top bar
When we push the idea further it (the red square) should be a menu where clicking on it gives you the "Activities" and you can easily pick one.

_Originally posted by @fradarr in https://github.com/wintermeyer/omedis/discussions/270#discussioncomment-11212041_

### PR DESCRIPTION
**Pull request by amos-kibet**
Resolves #317.

https://github.com/user-attachments/assets/8a457d75-ff05-473f-9d66-fb05476221cb

https://github.com/user-attachments/assets/7dd92cd1-7d6c-45bf-98a6-9e9c62815189

Pending items:

- [x] Tests
- [ ] Potential refactoring

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -24,14 +24,15 @@ import topbar from "../vendor/topbar";
 
 import Alpine from "alpinejs";
 
-import FlashAutoDisappear from "./hooks/flash_auto_disappear"
-import SlugInput from "./hooks/slug_input"
+import FlashAutoDisappear from "./hooks/flash_auto_disappear";
+import HideOnNavigate from "./hooks/hide_on_navigate";
```
Comment (amos-kibet) [inline assets/js/app.js:28]: Bring back `SlugInput` hook import.

File: lib/omedis_web/live/time_tracker_live/index.ex [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+
+  def handle_event("load-more-activities", _params, socket) do
+    {:noreply, maybe_assign_activities(socket)}
+  end
+
+  defp create_event(activity_id, opts) do
+    {:ok, activity} = Activity.by_id(activity_id, opts ++ [load: [:events]])
+
+    {:ok, _event} =
+      Event.create(
```
[the remark this answers is not in the document]
Comment (amos-kibet) [reply lib/omedis_web/live/time_tracker_live/index.ex:354]: This was the case, thanks!

File: assets/js/app.js (hunk 1 above)
Comment (almirsarajcic) [reply assets/js/app.js:28]: Hahahah
Did you just impersonate me :D

File: lib/omedis_web/live/time_tracker_live/index.ex (hunk 2 above)
[the remark this answers is not in the document]
Comment (almirsarajcic) [reply lib/omedis_web/live/time_tracker_live/index.ex:408]: Got you

**Review by almirsarajcic — APPROVED** [review]

**Pull request merged**